.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [main]
 
 env:
-  OLDEST_PYMC_VERSION: "5.7.1"
+  OLDEST_PYMC_VERSION: "5.8.2"
```

Comment:
Is htis actually required?

Reply:
The comment on the `.toml` file mentioned the following:

1. "NOTE: Keep minimum Pymc version in sync with ci.yml `OLDEST_PYMC_VERSION`"

As well, @ricardoV94 mentioned I should modify `pyproject` and the `ci` files.

Reply:
Yes this is needed for the CI